app/frontend/components/pages/login/loginPage.tsx

```diff
@@ -5,7 +5,7 @@ import isLeftClick from '../../../helpers/isLeftClick'
 
 import KeyFileAuth from './keyFileAuth'
 import MnemonicAuth from './mnemonicAuth'
-import HardwareAuth from './hardwareAuth'
+// import HardwareAuth from './hardwareAuth'
```

Comment:
I would just remove the line completely, we have it git history anyway and whoever will need to reenable the functionality will need to do probably a lot more than just uncommenting this line

Reply:
Yes, you are right, someone reenabling this functionality would have to to a lot more than just commenting these lines, but not in this file. This file most likely wont change at all, just the implementation of the crypto providers, so that's the reason I only commented these lines. But anyway, we can always get them from git history, so I removed the lines related to hardware wallets.